docs/getting-started/docker-playgrounds/flink_cluster_playground.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Flink Cluster Playground"
+nav-title: 'Flink Cluster Playground'
+nav-parent_id: docker-playgrounds
```

Comment:
https://fontawesome.com/icons/docker?style=brands ?

Reply:
I did not want to get into this: "This work includes material that may be protected as a trademark in some jurisdictions. If you want to use it, you have to ensure that you have the legal right to do so and that you do not infringe any trademark rights.
The use of these trademarks does not indicate endorsement of the trademark holder by Font Awesome, nor vice versa.
See the trademark owner for rules about appropriate use of their trademarks."

See also: https://github.com/wesbos/Font-Awesome-Docker-Icon. I am not comfortable in using it.